core/bam_core/utils/geo.py

```diff
@@ -67,6 +68,9 @@ def format_address(
     # common address mistakes/translations
     address = _fix_address(address)
 
+    # get plus code from GoogleMaps util
+    response["plus_code"] = gmaps.get_pluscode(address_query)
+
```

Comment:
This will fail because `address_query` isn't defined until the line below. So I think it would be:

```suggestion
    # get plus code from GoogleMaps util
    response["plus_code"] = gmaps.get_pluscode(address)
```

However, we might also consider doing address normalization first and then computing the `plus_code`. Though I have also seen incorrect results from address normalization, so I'm not sure which is better. Regardless, there are going to be some incorrect codes generated because the input addresses are super messy.

Reply:
yeah meant to do it after normalization, just copied the lines into the wrong place during editing